playbooks/ae/setup_ae_elk.yml

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2020 Cable Television Laboratories, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- hosts: ae
+  gather_facts: no
+  tasks:
+    - name: "Start Elasticsearch service"
+      become: yes
+      shell : service elasticsearch start
+
+    - name: "Start Kibana service"
+      become: yes
+      shell : service kibana start
+
+    - name: "Validate if a DNS tps.sdn.org is present in /etc/hosts file"
+      become: yes
+      tags: sdnDnsName
+      lineinfile:
+        path: /etc/hosts
+        regexp: "tps.sdn.org"
+        line: "tps.sdn.org {{sdn_ip}}"
+        state: present
+        backup: yes
+      register: sdnDnsNameOut
+
+    - name: POST call to create data mapping template for incoming data in Elasticsearch
+      uri:
+        url: "http://{{ae_ip}}:9200/_template/packets"
+        method: PUT
+        return_content: yes
+        body: "{{ lookup('template','udp_data_mapping.json.j2') }}"
```

Comment:
You may just want to use the file contents rather than using a template; however, keeping it as-is would give us more flexibility down-the-road.